apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorOptionsSection.vue

```diff
@@ -4,52 +4,149 @@
+                      :disabled="state.isUpdating"
+                      @click="handleClickCancel"
+            >
+                {{ $t('INVENTORY.COLLECTOR.DETAIL.CANCEL') }}
+            </p-button>
+            <p-button v-if="state.isEditMode"
+                      style-type="primary"
+                      size="lg"
+                      :loading="state.isUpdating"
+                      class="save-changes-button"
+                      @click="handleClickSave"
+            >
+                {{ $t('INVENTORY.COLLECTOR.DETAIL.SAVE_CHANGES') }}
+            </p-button>
+        </div>
     </p-pane-layout>
 </template>
 
 <script lang="ts" setup>
-import { defineProps, computed, reactive } from 'vue';
+import { computed, reactive } from 'vue';
 
 import {
     PHeading, PButton, PPaneLayout, PDefinitionTable,
 } from '@spaceone/design-system';
 import type { DefinitionField } from '@spaceone/design-system/types/data-display/tables/definition-table/type';
+import type { JsonSchema } from '@spaceone/design-system/types/inputs/forms/json-schema-form/type';
+
+import { i18n } from '@/translations';
+
+import { showSuccessMessage } from '@/lib/helper/notice-alert-helper';
+
+import ErrorHandler from '@/common/composables/error/errorHandler';
 
-import type { CollectorPluginModel } from '@/services/asset-inventory/collector/model';
+import type { CollectorModel, CollectorOptions } from '@/services/asset-inventory/collector/model';
+import { useCollectorFormStore } from '@/services/asset-inventory/collector/shared/collector-forms/collector-form-store';
+import CollectorOptionsForm from '@/services/asset-inventory/collector/shared/collector-forms/CollectorOptionsForm.vue';
 
-const props = defineProps<{
-    loading: boolean;
-    collectorOptions: CollectorPluginModel['options']|null;
-}>();
 
 
+const collectorFormStore = useCollectorFormStore();
+const collectorFormState = collectorFormStore.$state;
+
 const state = reactive({
+    loading: computed<boolean>(() => !collectorFormState.originCollector),
+    collectorOptions: computed<CollectorOptions>(() => collectorFormState.originCollector?.plugin_info?.options ?? {}),
+    collectorOptionsSchema: computed<JsonSchema>(() => collectorFormState.originCollector?.plugin_info?.metadata?.options_schema ?? {
+        type: 'object',
+        properties: {},
+    }),
     fields: computed<DefinitionField[]>(() => {
-        if (!props.collectorOptions) return [];
-        return Object.keys(props.collectorOptions).map((key) => ({
+        const properties = state.collectorOptionsSchema?.properties ?? {};
+        return Object.entries<JsonSchema['properties']>(properties).map(([key, property]) => ({
             name: key,
-            label: key,
+            label: property.title ?? key,
         }));
     }),
+    isEditMode: false,
+    isOptionsValid: false,
+    isUpdating: false,
 });
 
+const fetchCollectorUpdate = async (): Promise<CollectorModel> => {
+    // TODO: change to call api
```

Comment:
shock: No api...? yet...?😨

Reply:
No, This is the intend for splitting PRs :)